perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -162,6 +162,12 @@ sub munge_report_new_contacts {
 
 }
 
+sub munge_mixed_category_groups {
+    my ($self, $list) = @_;
+    my $nh = FixMyStreet::Cobrand::HighwaysEngland->new({ c => $self->{c} });
+    $nh->national_highways_cleaning_groups($list);
+}
```

Comment:
This replaces the cobrand-in core code - we need this both here and in UKCouncils to run everywhere it should.